environments/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "test",
+  "description": "Delivery Cluster Environment",
+  "json_class": "Chef::Environment",
+  "chef_type": "environment",
+  "override_attributes": {
+    "delivery-cluster": {
+      "accept_license": true,
+      "id": "test",
+      "driver": "openstack",
```

Comment:
setting ["delivery-cluster"]["driver"] to "openstack" allows us to load the lib/openstack.rb from this cookbook